frontend/www/js/omegaup/grader/ephemeral.js

```diff
@@ -350,6 +354,15 @@ let store = new Vuex.Store({
         ];
       document.getElementById('language').value = state.request.language;
     },
+    showSubmitButton(state, value) {
+      state.problemsetId = value;
+      if (value) {
+        const submitButton = document.querySelector(
+          'button[data-submit-button]',
+        );
+        submitButton.classList.remove('d-none');
+      }
```

Comment:
```suggestion
      const submitButton = document.querySelector(
        'button[data-submit-button]',
      );
      if (value) {
        submitButton.classList.remove('d-none');
      } else {
        submitButton.classList.add('d-none');
      }
```

siempre hay que permitir agregar Y quitar las clases depende de lo que nos manden.